pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -469,6 +469,9 @@ type ShootNodeLogging struct {
 
 // Logging contains configuration for the logging stack.
 type Logging struct {
+	// Enabled is used to enable or disable logging stack for clusters.
+	// +optional
+	Enabled *bool
```

Comment:
json/yaml tags are missing

Reply:
Addressed in 8c9ee3ad536361a54bdf56825cd1cb78a2bc8b72.